src/app/shared/components/template/pipes/qr-code.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import QRCode from "qrcode";
+
+@Pipe({
+  name: "qrCode",
+})
+export class QRCodePipe implements PipeTransform {
+  async transform(value: any, args?: any[]): Promise<any> {
```

Comment:
nice idea to have as a pipe, will make it easier to be responsive. The only downside is harder to defer loading, but that's an issue for another day